cmd/brew-test-bot.rb

```diff
@@ -179,19 +179,19 @@ def puts_command
         puts "travis_fold:start:#{@travis_fold_id}"
         puts "travis_time:start:#{@travis_timer_id}"
       end
-      puts "#{Tty.blue}==>#{Tty.white} #{@command.join(" ")}#{Tty.reset}"
+      puts Formatter.headline(@command.join(" "), color: :blue)
```

Comment:
You know, I still don't even know what `ohai` means, so let's leave it as-is for now. 😄